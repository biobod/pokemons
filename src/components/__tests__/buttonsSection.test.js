import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

import Adapter from 'enzyme-adapter-react-16';

import ButtonsSection from '../buttonsSection';

Enzyme.configure({adapter: new Adapter()});


describe('ButtonsSection', () => {
  it('should render correctly123', () => {
    const output = shallow(
      <ButtonsSection
        showPreviousPokemon={() => {}}
        showNextPokemon={() => {}}
      >
        <div>hello</div>
      </ButtonsSection>
    );
    expect(shallowToJson(output)).toMatchSnapshot();
  });
});
