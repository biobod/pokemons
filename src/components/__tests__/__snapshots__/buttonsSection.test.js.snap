// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ButtonsSection should render correctly123 1`] = `
<ButtonsSection
  classes={
    Object {
      "buttonSection": "ButtonsSection-buttonSection-2",
      "dashboardWrapper": "ButtonsSection-dashboardWrapper-1",
    }
  }
  showNextPokemon={[Function]}
  showPreviousPokemon={[Function]}
>
  <div>
    hello
  </div>
</ButtonsSection>
`;
